selfdrive/car/toyota/carcontroller.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, dbc_name, CP, VM):
     self.last_standstill = False
     self.standstill_req = False
     self.steer_rate_counter = 0
+    self.pcm_cancel_frame = 0
```

Comment:
```suggestion
    self.pcm_cancel_frame = -EXTRA_CANCEL_FRAMES
```

Make a global constant